cloud-functions/src/notification-engine/townhall/_utils/types.ts

```diff
@@ -104,6 +106,12 @@ export interface ITHNotification {
 	url?: string;
 }
 
+export enum EContentType {
```

Comment:
types should have the source project's Initials as prefix
her: `EContentType` -> `ETHContentType`